### PR TITLE
Updates for Forms 10.5.6, 13.2.0 and 14.1.0

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

This will include release notes and other updates for the next minor and patch releases of Forms.  It's now pretty much complete so ready for review.

Includes/to do:

- [x] Details of repeatable strings setting field https://github.com/umbraco/Umbraco.Forms.Issues/issues/1217
- [x] Additional placeholder formatter functions https://github.com/umbraco/Umbraco.Forms.Issues/issues/1205
- [x] SupportsHtml setting attribute
- [x] Additional data for workflows https://github.com/umbraco/Umbraco.Forms.Issues/issues/578
- [x] Release notes for 13.2 and 14.1
- [x] Details of text with field picker setting
- [x] Fix for local links in rich text email fields https://github.com/umbraco/Umbraco.Forms.Issues/issues/1208
- [X] Body text for email workflow https://github.com/umbraco/Umbraco.Forms.Issues/issues/1198
- [x] Fix for date field blur https://github.com/umbraco/Umbraco.Forms.Issues/issues/1241
- [x] Release notes for patch releases on earlier major versions
- [ ] Dates on release notes

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [X] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Forms 10.5.6, 13.2.0 and 14.1.0.

## Deadline (if relevant)

This is ready for review now, but please don't publish yet until other things are prepared and we can commit to a date.  Thanks.

### CODE REVIEW

File: 10/umbraco-forms/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ This section contains the release notes for Umbraco Forms 8 and 10 including all
 
 <summary>Version 10</summary>
 
+[**10.5.6**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F10.5.6) **(TBC)**
+
+* Fixed issue with save button UI when save is cancelled via a notification [#1219](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1219).
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:26]:
```suggestion
* Fixed issue with save button UI, when save is canceled via a notification [#1219](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1219).
```

File: 10/umbraco-forms/release-notes.md [hunk 2]
```diff
@@ -21,6 +21,13 @@ This section contains the release notes for Umbraco Forms 8 and 10 including all
 
 <summary>Version 10</summary>
 
+[**10.5.6**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F10.5.6) **(July 9th 2024)**
+
+* Fixed issue with save button UI when save is cancelled via a notification [#1219](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1219).
+* Improved date format for data values when using the "send email" workflow [#1214](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1214).
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:27]:
```suggestion
* Improved date format for data values when using the **Send email** workflow [#1214](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1214).
```

File: 13/umbraco-forms/developer/ajaxforms.md [hunk 3]
```diff
@@ -52,6 +52,8 @@ An optional `contentId` parameter can be provided, which can either be the integ
 
 A `culture` parameter can also be provided, expected as an ISO code identifying a language used in the Umbraco installation (for example, `en-US`). This will be used to ensure the correct translation for dictionary keys is used. It will also retrieve page content from the appropriate language variant. If the parameter is not provided in the request, the default Umbraco language will be used.
 
+Finally an `additionalData` parameter can be provided in the form of a dictionary. This information will be made available when rendering the form allowing it to be used as a source for ["magic string" replacements](./magic-strings.md).
```
Comment (eshanrnh) [inline 13/umbraco-forms/developer/ajaxforms.md:55]:
```suggestion
Finally, an `additionalData` parameter can be provided as a dictionary. This information will be made available when rendering the form allowing it to be used as a source for ["magic string" replacements](./magic-strings.md).
```

File: 13/umbraco-forms/developer/ajaxforms.md [hunk 4]
```diff
@@ -398,14 +400,20 @@ It also requires a `Content-Type` header of `application/json` and accepts a bod
...
+        "baz": "buzz",
+    }
 }
 ```
 
 The `values` collection consists of a set of name/value pairs, where the name is the alias of a form field. The value is the value of the submitted field, which can either be a string, or an array of strings. In this way we support fields that accept multiple values, such as checkbox lists.
 
 The `contentId` and `culture` parameters are optional. If provided they will be used to customize the response for the current page and language respectively.
 
+Similarly the `additionalData` dictionary is optional. This data is associated with the created record and made available within workflows.
```
Comment (eshanrnh) [inline 13/umbraco-forms/developer/ajaxforms.md:415]:
```suggestion
Similarly, the `additionalData` dictionary is optional. This data is associated with the created record and made available within workflows.
```

File: 13/umbraco-forms/developer/extending/setting-types.md [hunk 5]
```diff
@@ -20,11 +20,12 @@ The following setting types are available and are used for the field, prevalue s
...
 | Checkbox                  | Uses a single checkbox for entry                                  |                                                 |
-| DocumentMapper            | Used for selection of a documenttype                             | The "Save as Umbraco node" workflow             |
+| DocumentMapper            | Used for selection of a documenttype                              | The "Save as Umbraco node" workflow             |
 | Dropdownlist              | Used for selection from a list of options                         |                                                 |
 | EmailTemplatePicker       | Used for selection of an email template                           | The "Send email with Razor template" workflow   |
 | FieldMapper               | Used to map fields from a form to required aliases                | The "Send to URL" workflow                      |
 | File                      | Used for selection of a file                                      | The "Send email with XSLT template" workflow    |
+| MultipleTextString        | Used multiple textboxes for multiple entries                      | Not used in core types                          |
```
Comment (eshanrnh) [inline 13/umbraco-forms/developer/extending/setting-types.md:28]:
```suggestion
| MultipleTextString        | Uses multiple text boxes for entry                       | Not used in core types                          |
```

File: 13/umbraco-forms/release-notes.md [hunk 6]
```diff
@@ -17,6 +17,30 @@ If you are upgrading to a new major version, you can find information about the
 
 This section contains the release notes for Umbraco Forms 13 including all changes for this version.
 
+#### [**13.2.0-rc1**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.2.0) **(July 9th 2024)**
+
+* Added setting option for multiple and choice choice fields to allow for vertical or horizontal display [#1218](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1218).
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:22]:
```suggestion
* Added setting option for checkbox and radiobutton fields to allow for vertical or horizontal display [#1218](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1218).
```

File: 13/umbraco-forms/release-notes.md [hunk 7]
```diff
@@ -17,6 +17,30 @@ If you are upgrading to a new major version, you can find information about the
...
+* Allowed for the provision of additional data when rendering and submitting forms. When provided it will be used as a source for ["magic string" replacements](./magic-strings.md). The data will be associated with the created record and made available for custom logic and update within workflows. [#578](https://github.com/umbraco/Umbraco.Forms.Issues/issues/578).
+* Added details of workflow type to edit workflow dialog [#1183](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1183).
+* Removed an inline prevalue editor that wasn't functional but could surface it's UI when creating forms from templates [#1230](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1230).
+* Ensured local links are parsed when HTML fields are returned in the delivery API results for form definitions [#1227](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1227).
+* Resolved duplicate approval occurring when record is approved via a workflow [#1223](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1223).
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:29]:
```suggestion
* Resolved duplicate approval occurring when the record is approved via a workflow [#1223](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1223).
```

File: 13/umbraco-forms/release-notes.md [hunk 8]
```diff
@@ -17,6 +17,30 @@ If you are upgrading to a new major version, you can find information about the
...
+* Updated themes such that accessibility is improved by having hidden labels remain in markup but be visually hidden [#1220](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1220).
+* Fixed issue with save button UI when save is cancelled via a notification [#1219](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1219).
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:31]:
```suggestion
* Fixed issue with save button UI, when save is canceled via a notification [#1219](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1219).
```

File: 13/umbraco-forms/release-notes.md [hunk 9]
```diff
@@ -17,6 +17,30 @@ If you are upgrading to a new major version, you can find information about the
...
+* Improved date format for data values when using the "send email" workflow [#1214](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1214).
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:32]:
```suggestion
* Improved date format for data values when using the **Send email** workflow [#1214](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1214).
```

File: 13/umbraco-forms/release-notes.md [hunk 10]
```diff
@@ -17,6 +17,30 @@ If you are upgrading to a new major version, you can find information about the
...
+* Added setting option for the "send to URL" workflow to switch between caption and alias for the field value's XML element [#1202](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1202)
+* Allowed for use of prevalue sources that customize based on the current form or field in backoffice editing and preview [#1221](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1221).
+* Restored target used to generate local configuration schema information [#1226](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1226).
+* Fixed issues with retrieving fields and assigning default values when using the datasource wizard.
+* Ensured links to Umbraco pages within rich text fields used for emails are correctly parsed [#1208](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1208).
+* Added body rich text field for send email with Razor template workflow [#1198](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1198).
+* Fixed console error with blank values in data picker fields [#1241](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1241).
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:40]:
```suggestion
* Fixed console error with blank values in the date picker field [#1241](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1241).
```

File: 13/umbraco-forms/release-notes.md [hunk 11]
```diff
@@ -17,6 +17,30 @@ If you are upgrading to a new major version, you can find information about the
...
+* Ensured placeholders are parsed for accepted entry response from the delivery API [#1238](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1238).
+* Improved support for editing large, multi-page forms by retaining scroll position between views and adding a "jump to page" option [#1238](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1243).
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:42]:
```suggestion
* Improved support for editing large, multi-page forms by retaining scroll position between views and adding a "jump to page" option [#1243](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1243).
```

File: 14/umbraco-forms/developer/ajaxforms.md [hunk 12]
```diff
@@ -52,6 +52,8 @@ An optional `contentId` parameter can be provided, which can either be the integ
 
 A `culture` parameter can also be provided, expected as an ISO code identifying a language used in the Umbraco installation (for example, `en-US`). This will be used to ensure the correct translation for dictionary keys is used. It will also retrieve page content from the appropriate language variant. If the parameter is not provided in the request, the default Umbraco language will be used.
 
+Finally an `additionalData` parameter can be provided in the form of a dictionary. This information will be made available when rendering the form allowing it to be used as a source for ["magic string" replacements](./magic-strings.md).
```
Comment (eshanrnh) [inline 14/umbraco-forms/developer/ajaxforms.md:55]:
```suggestion
Finally, an `additionalData` parameter can be provided as a dictionary. This information will be made available when rendering the form allowing it to be used as a source for ["magic string" replacements](./magic-strings.md).
```

**Review by eshanrnh — APPROVED** [review]
Thanks for the PR, @AndyButland 🙌 I'll incorporate the minor comments/suggestions.

I just have 1-2 suggestions/queries that am not sure of. Can you take a look at it?

File: 13/umbraco-forms/release-notes.md (hunk 6 above)
Comment (AndyButland) [reply 13/umbraco-forms/release-notes.md:22]: Best would actually be "single and multiple choice fields" - as those are what we call them in the user interface.

File: 14/umbraco-forms/release-notes.md [hunk 13]
```diff
@@ -17,11 +17,29 @@ If you are upgrading to a new major version, you can find information about the
 
 This section contains the release notes for Umbraco Forms 14 including all changes for this version.
 
+#### [**14.1.0-rc1**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F14.1.0) **(July 9th 2024)**
+
+* Added setting option for multiple and choice choice fields to allow for vertical or horizontal display [#1218](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1218)
```
[the remark this answers is not in the document]
Comment (AndyButland) [reply 14/umbraco-forms/release-notes.md:22]: Best would actually be "single and multiple choice fields" - as those are what we call them in the user interface.

File: 14/umbraco-forms/release-notes.md [hunk 14]
```diff
@@ -17,11 +17,29 @@ If you are upgrading to a new major version, you can find information about the
...
+* Allowed for the provision of additional data when rendering and submitting forms. When provided it will be used as a source for ["magic string" replacements](./magic-strings.md). The data will be associated with the created record and made available for custom logic and update within workflows. [#578](https://github.com/umbraco/Umbraco.Forms.Issues/issues/578)
+* Added details of workflow type to edit workflow dialog [#1183](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1183)
+* Allowed for use of prevalue sources that customize based on the current form or field in backoffice editing and preview [#1221](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1221)
+* Ensured links to Umbraco pages within rich text fields used for emails are correctly parsed [#1208](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1208).
+* Added body rich text field for send email with Razor template workflow [#1198](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1198).
+* Fixed console error with blank values in data picker fields [#1241](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1241).
```
[the remark this answers is not in the document]
Comment (eshanrnh) [reply 14/umbraco-forms/release-notes.md:30]: Should this be date picker, @AndyButland?

[the remark this answers is not in the document]
Comment (AndyButland) [reply 14/umbraco-forms/release-notes.md:30]: It's fine how you have corrected it.

**Comment by eshanrnh** [comment]
I'll merge it in tomorrow 💪

**Pull request merged**